code/modules/mob/living/simple_animal/hostile/terror_spiders/actions.dm

```diff
@@ -137,6 +137,9 @@
 /mob/living/simple_animal/hostile/poison/terror_spider/proc/Web()
 	if(!web_type)
 		return
+	if(!isturf(loc))
+		to_chat(src, "<span class='danger'>Webs can only be spun while standing on a floor.</span>")
```

Comment:
the span class `warning` is usually used for these messages so they don't spam up the chat to much but are still noticeable

Reply:
They can't really spam the chat unless the user deliberately spam-clicks the web button while they're inside a pipe or similar.
In that rare case, I want it to be obvious so they learn immediately that they can't do it.